src/main/java/seedu/address/model/person/assignment/initialise/AssignmentNameInitialise.java

```diff
@@ -49,6 +50,10 @@ public AssignmentName getName(int index) {
         return assignmentNames.get(index);
     }
 
+    public static ObservableList<AssignmentName> getAllNames() {
+        return assignmentNames;
+    }
+
     public boolean contains(AssignmentName assignmentName) {
```

Comment:
wait, this method seems redundant. Not sure if i am high... but...
contain contains.